File: src/main/kotlin/org/kite9/diagram/common/elements/Routable.java
package org.kite9.diagram.common.elements;



/**
 * The {@link SSPEdgeRouter} requires that vertices implement Routeable so that it can route edges 
 * correctly around them.
 * 
  * @author robmoffat
 *
 */
public interface Routable {

	public RoutingInfo getRoutingInfo();
	
	public void setRoutingInfo(RoutingInfo gi);

}

File: src/main/kotlin/org/kite9/diagram/common/elements/Routable.kt
package org.kite9.diagram.common.elements

/**
 * The [SSPEdgeRouter] requires that vertices implement Routable so that it can route edges
 * correctly around them.
 *
 * @author robmoffat
 */
interface Routable {
    var routingInfo: RoutingInfo
}